styleguide/backstop.json

```diff
@@ -107,6 +107,19 @@
       ],
       "misMatchThreshold" : 1,
       "requireSameDimensions": false
+    },
+    {
+      "label": "topics-page",
+      "url": "http://localhost:3000/?p=pages-topic",
```

Comment:
Please use the open in new window URL and update your tests, some of your screenshots are shorter than the actual page.